libs/application/types/src/lib/Fields.ts

```diff
@@ -128,6 +129,10 @@ export type TableRepeaterItem = {
       currency?: boolean
       suffix?: string
     }
+  | {
+      component: 'phone'
+      format?: string
+    }
```

Comment:
No need to have the `format` prop, as the PhoneInput handles formatting the number automatically based on country code.

It would however be useful to be able to change these props:
```typescript
allowedCountryCodes?: string[]
enableCountrySelector?: boolean 
```

Also preferably the `enableCountrySelector`prop should default to false (this is an opt-in feature in the application-system)